atomic_reactor/plugins/koji_import.py

```diff
@@ -801,3 +803,19 @@ def _update_build(self, build):
             'release': nvr['release'],
             'source': self.workflow.data.koji_source_source_url,
         })
+
+
+def escape_non_printable_chars(koji_metadata: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Don't we usually define functions at the top before classes?

Reply:
moved